static/css/perdiem.css

```diff
@@ -327,6 +323,10 @@ ul.tabs li {
     background-color: #333333;
 }
 
+.tabs-content {
+    border: none;
+}
```

Comment:
FYI @localastronaut I added this CSS to remove an ugly border added by Foundation which was noticeable on the new Feedback page (I noticed it's also visible on the Change Password page).